airbyte-commons-worker/src/main/java/io/airbyte/workers/config/ApiClientBeanFactory.java

```diff
@@ -53,6 +54,8 @@ public ApiClient apiClient(
         .setPort(parsePort(airbyteApiHost))
         .setBasePath("/api")
         .setHttpClientBuilder(HttpClient.newBuilder().version(Version.HTTP_1_1))
+        .setConnectTimeout(Duration.ofSeconds(30))
```

Comment:
nit: can we define these as 'CONNECT_TIMEOUT_DURATION` and `READ_TIMEOUT_DURATION` constants?

Reply:
I didn't see this comment - I made the fix in the next PR